### ISSUE TITLE
QNAP Install broken after auto update

### ISSUE THREAD

**Issue by GTunney** [SickChill/SickChill#5154]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: Master
OS: QTS 4.3.5
Browser: Multiple
What you did: Auto update
What happened: SickChill Restart but did not come back up
What you expected: Sickchill to come back up
Logs:
```
[/share/CACHEDEV1_DATA/.qpkg/QSickChill] # /share/CACHEDEV1_DATA/.qpkg/QSickChill/QSickChill.sh start
file exists
Traceback (most recent call last):
  File "SickBeard.py", line 69, in <module>
    import sickbeard
  File "/share/CACHEDEV1_DATA/.qpkg/QSickChill/SickChill/sickbeard/__init__.py", line 46, in <module>
    from sickbeard import (auto_postprocessor, dailysearcher, db, helpers, logger, metadata, naming, post_processing_queue, properFinder, providers, scheduler,
  File "/share/CACHEDEV1_DATA/.qpkg/QSickChill/SickChill/sickbeard/dailysearcher.py", line 27, in <module>
    from sickbeard import common, db, logger, network_timezones
  File "/share/CACHEDEV1_DATA/.qpkg/QSickChill/SickChill/sickbeard/db.py", line 34, in <module>
    from sickbeard import logger
  File "/share/CACHEDEV1_DATA/.qpkg/QSickChill/SickChill/sickbeard/logger.py", line 43
    from requests.compat quote
                             ^
SyntaxError: invalid syntax
[/share/CACHEDEV1_DATA/.qpkg/QSickChill] #
```

SickChill auto updated yesterday about 14:15 and after the auto update restart it did not come back up. When I try to manually start via SSH I get the above. Not too sure what has happened, was something pushed that shouldn't have been?

Thanks,
Greg

**Comment by jxs** [comment]
also had an error similar to this regarding syntax, managed to solve by manually git pulling latest changes from origin master

**Comment by belinux** [comment]
Fixed in ec4903128d7674a334873835ba35d72b9991a924 
By pulling it will be fixed.

**Comment by GTunney** [comment]
Perfect thanks very much, I'm back up and running.

I thought Lord Voldermort had been messing again with all his RAGE ;-)
